spec/line_spec.rb

```diff
@@ -22,4 +22,17 @@
       expect(line.time_str).to eq("00:03:44,200 --> 00:04:04,578")
     end
   end
+
+  describe "#to_s" do
+    let(:line) { SRT::Line.new }
+
+    before do
+      line.sequence = "1"
+      line.start_time = 224.2
+      line.end_time = 244.578
+    end
+    it "should produce timecodes that match the internal float values" do
```

Comment:
should this spec title be something like `it "creates valid nodes with empty content"`?

Reply:
or really more of a:

```
describe "#to_s" do
  context "with empty content" do
    it "creates a valid empty node" do
      ...
    end
  end
end
```